src/styles/styles.js

```diff
@@ -374,9 +374,9 @@ const styles = {
     buttonMedium: {
         borderRadius: variables.buttonBorderRadius,
         height: variables.componentSizeNormal,
-        paddingTop: 6,
+        paddingTop: 12,
         paddingRight: 16,
-        paddingBottom: 6,
+        paddingBottom: 12,
```

Comment:
@shawnborton Can you please confirm the medium button vertical padding?

Reply:
I think this should be good. We could test quickly by having a letter in the button that has an descender, like "Again" so we can make sure that the bottom part of the g isn't cut off.

Reply:
on it! 
<img width="579" alt="image" src="https://user-images.githubusercontent.com/38015950/203608828-b5342794-0b04-49c3-9fe4-3b8119a7604a.png">

Reply:
Looks good!